src/utils/FitnessLevel.ts

```diff
@@ -0,0 +1,57 @@
+import stats, { type Section } from './stats';
```

Comment:
I see, so you just put all the functions in here and got rid of the class shit. Good idea 👍